pennylane_sf/remote.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Strawberry Fields Hardware device
+=================================
+
+**Module name:** :mod:`pennylane_sf.hw`
+
+.. currentmodule:: pennylane_sf.hw
+
+The Strawberry Fields Hardware plugin implements all the :class:`~pennylane.device.Device` methods,
+and provides access to Xanadu's continuous-variable quantum hardware.
+
+Classes
+-------
+
+.. autosummary::
+   StrawberryFieldsRemote
+
+
+Code details
+~~~~~~~~~~~~
+"""
```

Comment:
(Note: since the plugin docs refactor by Maria, I don't think these module docstrings show up in the docs anymore. So you could remove the 'Classes', 'Code details' sections, module directives, and the autosummary table)

Reply:
Thank you! Removed these from the file